src/reducers/settings.js

```diff
@@ -86,6 +87,7 @@ const INITIAL_STATE: SettingsState = {
   experimentalUSBEnabled: false,
   countervalueFirst: false,
   hideEmptyTokenAccounts: false,
+  blacklistedTokenIds: ["ethereum/erc20/0x_project"],
```

Comment:
why?

Reply:
Because I finished at 4 am and I wasn't thinking straight